Solution1/Solution1.Tests/BL/Validators/ForecastWeatherRequestDTOValidatorTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Vlidators;
+using FluentValidation;
+using FluentValidation.Results;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Weather.Tests.BL.Validators
+{
+    public class ForecastWeatherRequestDTOValidatorTests
+    {
+        private readonly IValidator<ForecastWeatherRequestDTO> _validator;
+
+        public ForecastWeatherRequestDTOValidatorTests()
+        {
+            _validator = new ForecastWeatherRequestDTOValidator();
+        }
+
+        [Theory]
+        [InlineData("Minsk", 3, false, true)]
```

Comment:
not all test cases are covered!
1) name rule set - ALL successful and unsuccessful paths
2) all rule set - ALL successful and unsuccessful paths (including name of course, cause in all rule set option validator verifies Name as well. Where do you check it?)

Reply:
+